### DISCUSSION TITLE
Trouble with starting a new game with Devouring&Digesting

### DISCUSSION THREAD

**Discussion by Maleccia** [Squidpad/BG3DevouringAndDigesting#31] [Q&A]
<img width="1018" alt="Screenshot 2024-11-04 003512" src="https://github.com/user-attachments/assets/bd2cf0ab-9031-4d3e-930b-9dc5016d3e17">
<img width="1064" alt="Screenshot 2024-11-04 005311" src="https://github.com/user-attachments/assets/ab66d174-d379-40cf-ba03-6ee2ef18d85b">

I have been unable to play with this mod in bg3 since september, I have updated everything, reinstalled everything, only with SSE(the most up-to-date version), up-to-date-mcm, and DevouringAndDigesting. However I keep getting this error. I tried other mods to see and they all seem to work fine, except for this one.
<img width="354" alt="Screenshot 2024-11-04 005731" src="https://github.com/user-attachments/assets/5ce10bd1-bae9-41e5-9fb1-7e3c37df7515">

What have I been doing wrong? Is the mod not playable in current patch?

**Comment by Mr12451** [comment]
I've heard that the old mod fixer could be the issue, so I removed it in the new release. See if it helps
